docs/content/doc/advanced/external-renderers.en-us.md

```diff
@@ -86,6 +86,11 @@ FILE_EXTENSIONS = .md,.markdown
 RENDER_COMMAND  = pandoc -f markdown -t html --katex
 ```
 
-You may redefine `ELEMENT`, `ALLOW_ATTR`, and `REGEXP` multiple times; each time all three are defined is a single policy entry. All three must be defined, but `REGEXP` may be blank to allow unconditional whitelisting of that attribute.
+You must define `ELEMENT`, `ALLOW_ATTR`, and `REGEXP` in each section.
+
+To define multiple entries, define different section names (e.g., `[markup.sanitizer.1]` and `[markup.sanitizer.2]`).
+These can be numbers, identifying names, or anything else.
 
 Once your configuration changes have been made, restart Gitea to have changes take effect.
+
+**Note**: The above section numbering policy is new; previously the section was `[markup.sanitizer]` and keys could be redefined.
```

Comment:
```suggestion
```

Same as above

Reply:
I had specifically requested some information as this is a breaking change requiring change to config (although the previous config simply does not work.)

Reply:
To avoid duplicating too much info, I've added a reference to the cheat sheet here but explicitly left this line. Its either that or add a reference to this pull request (in both places) if we don't want it in the documentation -- but to someone wanting to migrate, it isn't clear what the problem is just by looking at our extended discussion here. :-)

Reply:
> I had specifically requested some information as this is a breaking change requiring change to config (although the previous config simply does not work.)

Sorry, @zeripath. I missed your comment, which was already resolved.